### ISSUE TITLE
Translations in Portuguese/Spanish

### ISSUE THREAD

**Issue by miglsoares** [dlbnco/tripby.org#87]
# Feature request
Translations in Portuguese/Spanish can help a lot of people working with harm reduction informations. 7

PS: I can help with i18n!

**Comment by dlbnco** [comment]
@miglsoares 

I agree! I believe there is multi-language support in PsychonautWiki (all data comes from there) but I don't think they are exposed to the API.

Our messages though are already structured which makes it easier to translate and implement i18n:

https://github.com/dlbnco/tripby.org/blob/develop/components/IntlProvider/messages.js

**Comment by VerifiedGruber** [comment]
> @miglsoares
> 
> I agree! I believe there is multi-language support in PsychonautWiki (all data comes from there) but I don't think they are exposed to the API.
> 
> Our messages though are already structured which makes it easier to translate and implement i18n:
> 
> https://github.com/dlbnco/tripby.org/blob/develop/components/IntlProvider/messages.js

The API has a repository here on Github, an issue asking for the possibility of multiple languages should be a reasonable objective for a wiki-like website focused on aggregating accessible information. I'll look into [PsychonautWiki To-do list](https://psychonautwiki.org/wiki/To-do_list) and their API respectively and create an issue tagging you. Meanwhile, there are plugins that allow for automatic translation of JS API's, but I haven't researched them in-depth yet, still, they could provide a hotfix for the time being
